artisatomic/readqubdata.py

```diff
@@ -29,18 +29,38 @@ def read_adf04(filepath, flog):
         line = fleveltrans.readline()
         row = line.split()
         ionization_energy_ev = float(row[4].split("(")[0]) * hc_in_ev_cm
+        # Formatting of the calculation details section at the bottom of each file is not standardised
+        # This ignores it, provided it's put in the correct place. Needs made more robust later.
+        # Currently if it's somewhere random/random formatting: ¯\_(ツ)_/¯, delete manually.
+        atomic_group_note = False
         while True:
             line = fleveltrans.readline()
             if not line or line.startswith("   -1"):
                 break
-            config = line[5:21].strip()
+            if line.startswith("C-"):
+                atomic_group_note = not atomic_group_note
+                continue
+            if atomic_group_note:
+                continue
+            
+            config_full = line[5:27].strip()
+            # Accounting for files which have multiple sets of brackets in this string
+            # Some files include parent terms
+            config_parts = config_full.split(")")
+
+            if len(config_parts) > 1:
+                relevant_config = config_parts[-2].strip() + ")"
+            else:
+                relevant_config = config_full
+
+            config = relevant_config
             energylevel = qub_energy_level_row(
                 config,
                 int(line[:5]),
-                int(line[25:26]),
-                int(line[27:28]),
-                float(line[29:33]),
-                float(line[34:55]),
+                int(line[29:30]),
+                int(line[31:32]),
+                float(line[33:37]),
+                float(line[38:59]),
```

Comment:
I should probably check this myself, but are you sure this doesn't break reading in the existing Cobalt data from Tyndall? It seems as if it could.